Gemfile

```diff
@@ -1,5 +1,15 @@
-source "http://rubygems.org"
+# frozen_string_literal: true
+
+source 'http://rubygems.org'
 
 gemspec
 
+gem 'polishgeeks-dev-tools', '~> 1.4.0'
```

Comment:
Please remove this, it will be deprecated no need to add so many dependencies even in development. Let's use Coditsu instead.